sdk/storage/azure-storage-common/vcpkg/CONTROL

```diff
@@ -0,0 +1,7 @@
+Source: azure-storage-common-cpp
+Version: @AZ_LIBRARY_VERSION@
+Build-Depends: azure-core-cpp, libxml2, openssl (!windows)
```

Comment:
bcrypt, LibXML2

Reply:
libxml2 is listed, bcrypt is WinAPI and not vcpkg.